src/security/backups.md

```diff
@@ -0,0 +1,19 @@
+# Backup and restore
+
+Backups vary by our offering and their retention is governed by our [data retention](/security/data-retention.md) policies.
+
+Below are details on our Recovery Point Objective (RPO) and Recovery Time Objective (RTO) for our Platform Standard and Platform Enterprise offerings.
+
+## Platform Standard
+
+With Platform Standard, we have enabled our clients to manage their own backup and restore functions. Please see the [snapshot and restore](/administration/snapshot-and-restore.md) page for details.
+
+**RPO**: User defined. The RPO is configured by our clients.
+
+**RTO**: Variable. Recovery time will depend upon the size of the data we are recovering
```

Comment:
This is not accurate, AFAIK.  It is variable based on a whole bunch of factors like network and system load, but data size is the only thing it doesn't depend on.  (Engineering can confirm/dispute that.)

Reply:
Are you referencing RTO? This is the engineering vetted language that we've (myself & Rob) been using. The amount of data we need to restore definitely impacts how long it will take to perform the restore.

Reply:
For enterprise it's definitely also tied to data volume. If you have to recreate MySQL from a text based dump, the volume of data is linearly correlated to time.

Reply:
For Enterprise, definitely it's size-dependent.

I just checked with Engineering, though, and it is size dependent for PS restore, too, due to the need to copy the backup from S3 first.  So I withdraw my objection to this verbage.